man/docker-network-create.1.md

```diff
@@ -0,0 +1,55 @@
+% DOCKER(1) Docker User Manuals
+% Docker Community
+% OCT 2015
+# NAME
+docker-network-create - create a new network
+
+# SYNOPSIS
+**docker network create**
+
+[**-d**|**--driver**=[**name**]]     
+[**--help**]
+
+# DESCRIPTION
+
+Creates a new network. Driver accepts any of the following `name` values:
+
+
+| Value     | Description                                                                                                                                                                         |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `bridge`  | Uses Linux bridging and iptables to connect containers. This creates a single bridge, called `docker0` by default, and attaches a veth pair between the bridge and every container. |
+| `null`    | A noop implementation of the driver API, used only in cases where no networking is desired. This is to provide backward compatibility to the Docker's --net=none option.            |
+| `overlay` | Implements networking that can span multiple hosts using overlay network encapsulations such as VXLAN.                                                                              |
+| `remote`  | Not a driver but supports drivers over a remote transport. This allows you to [write a driver in the language of your choice](https://github.com/docker/libnetwork/blob/master/docs/remote.md).                                                       |
+
+The following command creates a network called `multi-host-network` managed by
+the `overlay` storage driver.
+
+```bash
+$ docker network create -d overlay multi-host-network
+```
+Network names must be unique across a cluster. The Docker daemon attempts to identify naming conflicts but this is not guaranteed. It is the user's responsibility to avoid name conflicts.
+
+When you start a container, use the `--net` flag to connect it to a network. This adds the `busybox` container to the `multi-host-network` network.
+
+```
+$ docker run -itd --net=multi-host-network busybox
+```
+If you want to add a container to a network after the container is already running, use the `docker network connect` subcommand. You can disconnect a container from a container using the `docker network disconnect` command.
+
+**MADHU: The first sentence below is odd. What does it mean for containers "to communicate" which each other? Is spontaneous? Also the note sounds like a "TODO" for the implementation. Was this fixed?**
+
+Multiple containers can be connected to the same network and the containers in the same network will start to communicate with each other. If the driver/plugin supports multi-host connectivity, then the containers connected to the same multi-host network will be able to communicate seamlessly.
+
+**Note**: UX needs enhancement to accept network options to be passed to the drivers
```

Comment:
@dave-tucker rc2 shows ---  ` -o, --opt=map[] `  are these two different things or was the flag changed on implementation --- `-o, --opt` is what the code in that PR shows.